docs/publish/publish.md

```diff
@@ -45,6 +45,12 @@ The deployment options that are available depend on the Office application that
 
 The following sections provide additional information about the deployment methods that are most commonly used to distribute production Office Add-ins to users within an organization.
 
+> [!IMPORTANT]
+> If you deploy updates to an already deployed add-in, some changes in the manifest require admin consent. Users are unable to use the updated add-in until consent is granted. The following changes in the manifest will require the admin to consent to them.
```

Comment:
Try this to avoid saying "deploy" in rapid succession.
```suggestion
> If you update an already deployed add-in, some changes in the manifest require admin consent. Users are unable to use the updated add-in until consent is granted. The following changes in the manifest require the admin to consent to them.
```

Reply:
fixed.